### ISSUE TITLE
Fiends Rework

### ISSUE THREAD

**Issue by Sullien** [Sullien/The-Demon-World#8]
Latest Push rebalances a bit the stats of fiends. 
Makes them one-time only by binding their spawns to unique wonders. Preventing White Rider spam.

Fiends's stats/costs are still a bit on the edge though, likely need further testing.

**Comment by heipizhu4** [comment]
Sorry for late reply, was sleeping when you made the PR. I' am going to test it right away.

**Comment by Sullien** [comment]
One of my ideas is to make them all have around the same base strength. And increase it a bit according to the era.

**Comment by heipizhu4** [comment]
> One of my ideas is to make them all have around the same base strength. And increase it a bit according to the era.

Still playing the game. It's a good idea but only problem is  fiend madetor and Demi-Fiend will have the same basestrength, which will be illogical. My idea is to upgrade them through eras with faith (Is it moddable?).

**Comment by heipizhu4** [comment]
And could we show the units in the civilopedia so they could be shown in the tech tree?

**Comment by Sullien** [comment]
> > One of my ideas is to make them all have around the same base strength. And increase it a bit according to the era.
> 
> Still playing the game. It's a good idea but only problem is fiend madetor and Demi-Fiend will have the same basestrength, which will be illogical. My idea is to upgrade them through eras with faith (Is it moddable?).

Well it doesn't need to be the exact same, just the bonuses. By upgrade you mean go from a fiend to another by paying faith? Also they could be shown. I kinda liked the aspect of keeping them as a hidden surprise but showing them might be more strategic.

**Comment by heipizhu4** [comment]
> By upgrade you mean go from a fiend to another by paying faith? 


Yup. (And it doesn't sound like a good idea as players can build a lower level of fiend and upgrade them, which causes imbalence.

> I kinda liked the aspect of keeping them as a hidden surprise but showing them might be more strategic.

Let's leave it for the other players to choose the one they like more. I'll advertise this mod on social media for a small poll after translation.

**Comment by Coolbrololer** [comment]
I think Demi-Fiend is too op for early eras.
Too much power that cities in the early eras deal no dmg, has Conceptual Existence which boost soo much culture and science that a great scientist is useless, let alone a great artist(i mean, it's quite fair as your only option to win faster is by doing so much war), and a lot more, just for a fiend soul, 0 faith, as long as you have your religion. (I was playing as The Demon World)
Overall, Too op for an early game, definitely gonna need some nerfs like it being available at industrial era, nerfing power to half or near it or it costing all Fiend souls(since 1 was already op enough, you can have as much as 4 while enemies are still at classical era)
Idk much about coding, but Demi-Fiend makes wars too easy and progression too easy.

**Comment by heipizhu4** [comment]
>  has Conceptual Existence which boost soo much culture and science that a great scientist is useless, let alone a great artist(i mean, it's quite fair as your only option to win faster is by doing so much war)

Do you think making Conceptual existence unique
```
"Earn [300]% of the damage done to [Military] units as [Science]","Earn [300]% of the damage done to [Military] units as [Culture]","[+10] HP when healing","Will not be displayed in Civilopedia"
```
into:
```
"Earn [100]% of the damage done to [Military] units as [Science]","Earn [100]% of the damage done to [Military] units as [Culture]","[+10] HP when healing","Will not be displayed in Civilopedia"
```
or even smaller is better?

**Comment by Sullien** [comment]
> I think Demi-Fiend is too op for early eras. Too much power that cities in the early eras deal no dmg, has Conceptual Existence which boost soo much culture and science that a great scientist is useless, let alone a great artist(i mean, it's quite fair as your only option to win faster is by doing so much war), and a lot more, just for a fiend soul, 0 faith, as long as you have your religion. (I was playing as The Demon World) Overall, Too op for an early game, definitely gonna need some nerfs like it being available at industrial era, nerfing power to half or near it or it costing all Fiend souls(since 1 was already op enough, you can have as much as 4 while enemies are still at classical era) Idk much about coding, but Demi-Fiend makes wars too easy and progression too easy.

Early eras? The building requires the Nuclear Fusion tech to be researched. You must have chosen a belief which lets you purchase units for Faith which is definitely an unintended interaction. They will no longer be purchasable by faith.

**Comment by Coolbrololer** [comment]
> > I think Demi-Fiend is too op for early eras. Too much power that cities in the early eras deal no dmg, has Conceptual Existence which boost soo much culture and science that a great scientist is useless, let alone a great artist(i mean, it's quite fair as your only option to win faster is by doing so much war), and a lot more, just for a fiend soul, 0 faith, as long as you have your religion. (I was playing as The Demon World) Overall, Too op for an early game, definitely gonna need some nerfs like it being available at industrial era, nerfing power to half or near it or it costing all Fiend souls(since 1 was already op enough, you can have as much as 4 while enemies are still at classical era) Idk much about coding, but Demi-Fiend makes wars too easy and progression too easy.
> 
> Early eras? The building requires the Nuclear Fusion tech to be researched. You must have chosen a belief which lets you purchase units for Faith which is definitely an unintended interaction. They will no longer be purchasable by faith.

Oooh so that's what it was. I was wondering why i should build a fiend wonder if i could just purchase it by faith, thanks for the fix! Will try it later.
